src/qt/transactiondesc.cpp

```diff
@@ -255,26 +285,34 @@ QString TransactionDesc::toHTML(interfaces::Node& node, interfaces::Wallet& wall
     strHTML += "<b>" + tr("Output index") + ":</b> " + QString::number(rec->getOutputIndex()) + "<br>";
 
     // Message from normal bitcoin:URI (bitcoin:123...?message=example)
-    for (const std::pair<std::string, std::string>& r : orderForm)
+    for (const std::pair<std::string, std::string>& r : orderForm) {
```

Comment:
This will change Message,Message,Merchant,Merchant into Message,Merchant,Message,Merchant.

Is that intentional? Probably should be in a separate commit...

Reply:
Does it really matter that much that the order is changed?

The only reason I changed it is because it looked funny that there were two loops in a row over the same vector.